test/integration/functional_test.go

```diff
@@ -783,7 +822,14 @@ func validateServiceCmd(ctx context.Context, t *testing.T, profile string) {
 		}
 	}()
 
-	rr, err := Run(t, exec.CommandContext(ctx, "kubectl", "--context", profile, "create", "deployment", "hello-node", "--image=k8s.gcr.io/echoserver:1.4"))
+	var echoServerArg string
+	if arm64Platform() {
```

Comment:
and why do we have two different echo server versions ?

if 1.10 is not availble in arm, how about making both to be 1.8

Reply:
Yeah, echoserver is not multiarch and the last available arm version was 1.8
Changed to 1.8 for all archs